frontend/src/lib/components/CommandBar/index.scss

```diff
@@ -0,0 +1,4 @@
+.command-bar__search-input {
```

Comment:
Should we be doing `.CommandBar__search-input` to stay in line with a lot of the newer styles we've added. I really shouldn't be calling out style given it is not linted...

Reply:
Yup, makes sense to stay consistent with BEM class names. Forgot we have the convention since I usually use the utility classes.